akka-persistence-query/src/main/scala/akka/persistence/query/journal/inmem/javadsl/InmemReadJournal.scala

```diff
@@ -1,39 +0,0 @@
-/*
```

Comment:
I'd like to get rid of it as well, but it looks like public API so we can't because of bincomp, have to stay with deprecation?

Reply:
Hrmm did i delete this on one of my commits? I meant to delete the read side that i added and now have removed in favor of adding one to the testkit